blog/models.py

```diff
@@ -19,7 +19,7 @@ class User(AbstractBaseUser):
     objects = UserManager()
 
     USERNAME_FIELD = 'email'
-    REQUIRED_FIELDS = []
+    REQUIRED_FIELDS = ['email', 'password', 'username']
```

Comment:
Yesterday I said that you only need to add username and email

Reply:
Done